embeddings/pipeline/sklearn_classification.py

```diff
@@ -0,0 +1,190 @@
+from typing import Tuple, Dict
+
+import datasets
+import numpy as np
+import pandas as pd
+from sklearn.feature_extraction.text import CountVectorizer, TfidfVectorizer
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import accuracy_score, precision_score, recall_score, f1_score
+
+
+class SklearnClassificationPipeline:
```

Comment:
Some of functionality here should be transfered to `SklearnModel`and `Sklearn Classification Task`. You can follow Flair examples for more details [FlairModel](https://github.com/CLARIN-PL/embeddings/blob/main/embeddings/model/flair_model.py#L12) and [Flair Classification Task](https://github.com/CLARIN-PL/embeddings/blob/main/embeddings/task/flair_task/text_classification.py#L13)